float32 range
float32 h_bearing
float32 v_bearing
float32[] data
string id
